### DISCUSSION TITLE
v1.2.7 顶栏 滚动时自动折叠失效

### DISCUSSION THREAD

**Discussion by ghost** [solstice23/argon-theme#318] [Q&A]
我的版本是v1.2.7
PHP版本 | 7.4.25
我的网站：https://derekdekker.com
顶栏 滚动时自动折叠 设置 失效，网站是新创建的，使用 Argon 顶栏一直没有正常折叠
谢谢您提供的主题用起来非常好。

**Comment by solstice23** [comment]
Bug report 发 issue 比较好
待修，是单栏下不会折叠
感谢反馈
